RealmSwift/Tests/RealmCollectionTypeTests.swift

```diff
@@ -904,90 +904,6 @@ class ListStandaloneRealmCollectionTypeTests: ListRealmCollectionTypeTests {
         assertThrows(collection.filter("ANY stringListCol == %@", CTTStringObjectWithLink()))
     }
 
-    override func testMin() {
```

Comment:
These tests were removed because this is now tested in Objective-C? I still think they're valuable here IMO.